css/_list.scss

```diff
@@ -0,0 +1,67 @@
+.list {
+  display: block;
+
+  @include breakpoint(md) {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &-wrapper {
+    margin-bottom: 2rem;
+
+    @include breakpoint(md) {
+      width: calc(calc(100% / 3) - 2rem);
```

Comment:
Is there a better way for this and avoid nested calc?

Reply:
I can make use of variables!
* --column-width 
* --wrapper-width

Or i can remove the inner calc ([stack overflow](https://stackoverflow.com/a/20732501)).
But i think I'll go for the variable method.

Also putting this here as a self note: https://css-tricks.com/a-complete-guide-to-calc-in-css/#aa-nesting-calccalc